ferrocene/doc/internal-procedures/src/manual-upstream-pulls.rst

```diff
@@ -11,96 +11,60 @@ An "upstream pull" is a pull request that pulls in commits from the upstream Rus
-  This is `main` if doing a `master` upstream pull and `release/1.70` for a
-  `beta` upstream pull.
-* `<UPSTREAM_BRANCH>`: The upstream branch to pull changes from. This is `master`
-  if doing a `master` upstream pull and `beta` for a `beta` upstream pull.
-* `<NEW_PULL_BRANCH>`: This represents the branch you will be making the pull
-  with. Replace it with a branch name of your choice, e.g.
-  `pull-upstream-master-2023-05-22`
+Manually resolving merge conflicts
+----------------------------------
 
-Initiating the pull
-^^^^^^^^^^^^^^^^^^^
+If an automatic upstream pull creates merge conflicts, the automation commits the
+merge conflicts and they need to be resolved manually.
 
-Run the following git commands to set up a new local `<NEW_PULL_BRANCH>`
-branch to do the pull on::
+Start by checking out the PR. The name of the branch is
+`automation/pull-upstream/<ID>`, e.g. `automation/pull-upstream/dfvjj2s4`::
 
-  git checkout <BASE_BRANCH> # check out the ferrocene branch to do the pull for
-  git pull --rebase # make sure your local branch is up to date
-  git submodule update # update any submodules that might've changed
-  git checkout -b <NEW_PULL_BRANCH> # checkout a new branch to do the pull with
+    git switch automation/pull-upstream/<ID>
 
-Performing the pull
-^^^^^^^^^^^^^^^^^^^
+In order to find the conflicts you can either look at the CI logs, or search for the
+git conflict markers (`<<<<<<<`, `=======` and `>>>>>>>`).
 
-Now run::
+There are multiple kinds of conflicts you will encounter:.
 
-  ferrocene/tools/pull-upstream/pull.sh <UPSTREAM_BRANCH>
-
-The script will fetch the latest changes from `rust-lang/rust`'s
-`<UPSTREAM_BRANCH>` branch, create a commit to remove the excluded files
-(see below) from the upstream branch, and finally try to merge upstream into
-the current branch.
-
-The script is able to resolve some kinds of merge conflicts on its own. If
-there are still remaining merge conflicts after the automatic resolution the
-script will exit. You'll have to fix the merge conflicts yourself in that case:
-
-There are two kinds of conflicts you will encounter in the output that the
-script produces:.
-
-* Submodule conflict ::
-
-    Please manually handle the merging of each conflicted submodule.
-    This can be accomplished with the following steps:
-    - go to submodule (src/tools/cargo), and either merge commit <COMMIT_HASH>
-      or update to an existing commit which has merged those changes
-    - come back to superproject and run:
-
-          git add src/tools/cargo
-
-      to record the above merge or update
-    - resolve any other conflicts in the superproject
-    - commit the resulting index in the superproject
-
-  The message already describes how to fix this conflict. The commands required
-  for this example are::
+* Submodule conflict
+  To fix a submodule conflict run::
 
     cd src/tools/cargo
     git merge <COMMIT_HASH>
     cd ../../..
     git add src/tools/cargo
```

Comment:
I don't think this will ever happen with the new way the automation works.

Reply:
Removed it